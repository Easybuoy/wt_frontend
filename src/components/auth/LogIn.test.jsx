import React from "react";
import { render } from "@testing-library/react";
import Login from "./LogIn";

describe("<Login/>", () => {
  it("should match snapshot", () => {
    const login = render.create(<Login />).toJSON();
    expect(login).toMatchSnapshot();
  });
});
