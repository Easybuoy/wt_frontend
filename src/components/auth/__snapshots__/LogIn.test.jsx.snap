// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogIn /> renders the LogIn component correctly 1`] = `
<ApolloConsumer>
  <Component />
</ApolloConsumer>
`;
